src/SDKs/Batch/DataPlane/Azure.Batch/GeneratedProtocol/JobScheduleOperations.cs

```diff
@@ -990,6 +990,10 @@ internal JobScheduleOperations(BatchServiceClient client)
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "this.Client.ApiVersion");
             }
+            if (jobSchedulePatchParameter == null)
+            {
+                jobSchedulePatchParameter = new JobSchedulePatchParameter();
```

Comment:
@matthchr how will this get hit? A few lines above you have

```cs
if (jobSchedulePatchParameter == null)
{
    throw new ValidationException(ValidationRules.CannotBeNull, "jobSchedulePatchParameter");
}
```

Reply:
@matthchr it looks like this comment can be applied to most null checks you are adding in the operation classes.

Reply:
I didn't add these checks, AutoRest did. This is in the generated client, generated from the latest AutoRest code generator (note the "GeneratedProtocol" folder)